shared/local-debug.desktop.js

```diff
@@ -39,6 +40,7 @@ let config: {[key: string]: any} = {
 
 if (__DEV__ && process.env.KEYBASE_LOCAL_DEBUG) {
   config.actionStatFrequency = 0.8
+  config.allowMultipleInstances = true
```

Comment:
Changing the flag requires a full webpack rebuild since these files are in the DLL, so it's kinda a pain to set this on a case-by-case basis. I think it makes sense to have this on by default when the debug env var is set, since I can't think of a reason the single-instance behavior would be desirable in dev (except to test that feature).